modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2LXCWithProvisioningRole.testcase

```diff
@@ -45,6 +45,62 @@ definition {
 		}
 	}
 
+	@description = "Story ID: LPS-153497 | Test Summary: Verify if you cancel the pop-up the card status remains the same"
+	@priority = 3
+	test AssertIfCloseTheModalStatusDontChange {
+		task ("Give the user has Admin and provisioning role in the project") {
+			JSONAccountRole.associateUser(
+				accountEntryName = ${testAccount},
+				accountRole = "Account Administrator",
+				userEmailAddress = "test@liferay.com");
+
+			JSONAccountRole.associateUser(
+				accountEntryName = ${testAccount},
+				accountRole = "Provisioning",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("And the user goes to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+
+			CustomerPortal.search(searchItem = ${testAccount});
+
+			CustomerPortal.enterOnProject(title = ${testAccount});
+		}
+
+		if (IsElementPresent(key_text = "Start Project Setup", locator1 = "Button#ANY")) {
```

Comment:
@styller1 could you add this block of code inside a task module?

Reply:
Done